src/main/resources/org/nest/sympy/integration_test.py

```diff
@@ -0,0 +1,42 @@
+from sympy import *
+import misc
+import ode_to_prop_matrix as otpm
+import unittest
+
+class TestShapesConversionAndPropagatorMatrxiComputation(unittest.TestCase):
+    def test_shape_conversion(self):
+        shape_alpha = misc.ShapeFunction("shape_alpha", "e / tau * t * exp(-t / tau)")
+        self.assertEqual(2, shape_alpha.order)
+
+        shape_exp = misc.ShapeFunction("shape_exp", "exp(-t/tau)")
+        self.assertEqual(1, shape_exp.order)
+
+        shape_1 = misc.ShapeFunction("shape_1",  "e / tau * t * t * exp(-t / tau)")
+        self.assertEqual(3, shape_1.order)
+        # shape_2 has order 4
+        # shape_2 = misc.ShapeFunction("shape_2",  "e / tau * t * t *t * exp(-t / tau)")
```

Comment:
I think, the reason was that I thought it didn't add any more information by multiplying with another $t$ (see shape alpha and shape 1). But then it makes more sense to delete it.